### PR TITLE
Improve Windows compatibility

### PR DESCRIPTION
**Pull request by jon-a-nygaard**
- Swapped use of `Path` to `Path.posix`
- Replace OS directory seperator with `/` on all inputted paths.

### CODE REVIEW

File: src/Config.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@
  * 
  *!*/
 
-import * as Path from 'path';
 import * as TSD from './TypeScriptDeclarations';
 import * as Utils from './Utilities';
+import { sep, posix } from 'path';
```
Comment (bre1470) [inline src/Config.ts]: Would be great, if you stick to alphabetical order.

File: src/NamespaceParser.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import * as Config from './Config';
 import * as Utils from './Utilities';
 import * as TSD from './TypeScriptDeclarations';
+import { sep, posix } from 'path';
```
Comment (bre1470) [inline src/NamespaceParser.ts]: Same here - alphabetical order, please.

File: src/Config.ts (hunk 1 above)
Comment (jon-a-nygaard) [reply src/Config.ts]: Sure thing 👍 Please see latest changes @bre1470

**Comment by bre1470** [comment]
Thank you! 👍

**Pull request merged**